scenarios/tracking/02_evaluating_mot_challenge.ipynb

```diff
@@ -0,0 +1,459 @@
+{
```

Comment:
<ol><li>base should probably be defined as MOTCHALLENGE_BASE_URL with the other constants in cell 3</li><li>consider making MOT_16_OR_17 an enum or something</li><li>consider adding some text explaining what <code>seqs_str</code>  is</li><li>print(f"Data saved to {MOT_SAVED_PATH}")</li></ol><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>